config.py

```diff
@@ -566,6 +566,14 @@
 # If the user passed thru extra defines, grab them
 if "DEFINES" in os.environ:
     DEFINES = os.environ["DEFINES"]
+# linker flags - LDFLAGS, SLDFLAGS
+if ("LDFLAGS" in os.environ):
+    LDFLAGS = os.environ["LDFLAGS"]
+if ("SLDFLAGS" in os.environ):
+    SLDFLAGS = os.environ["SLDFLAGS"]
+
+# Take the ones from the environment or command line,
```

Comment:
comments should be prior to what you are doing, would recommend putting the comments before line 570

Reply:
This comment seems duplicated from earlier in the code, line 560.  These comment lines here are unnecessary.

Reply:
sorry. it was copy paste mistake